src/views/LoginView.vue

```diff
@@ -35,15 +35,20 @@
         </p>
 
         <div class="icons mb-3">
-          <a :href="getOAuth2LoginUrl('google')" v-if="supportEmailAccounts">
-            <IconBase iconName="google" width="48" height="48" />
-          </a>
-          <a :href="getOAuth2LoginUrl('facebook')" :disabled="disableFacebook" v-if="supportEmailAccounts">
-            <IconBase iconName="facebook" width="48" height="48" />
-          </a>
-          <a :href="getOAuth2LoginUrl('microsoft')" :disabled="disableMicrosoft" v-if="supportEmailAccounts">
-            <IconBase iconName="microsoft" width="48" height="48" />
-          </a>
+
+          <!-- oauth2 login buttons -->
+          <span v-if="supportEmailAccounts">
```

Comment:
You can use `<template>` if you don't want a random `<span>` on the page instead.

Reply:
Oh hey, that's neat!

### _The More You Know ≈≈≈≈≈★_